0001-typedb/02-dev/05-read.md

```diff
@@ -0,0 +1,359 @@
+
+## Get query
+
+A get query is used to extract data instances or types (schema concepts) out of a database by describing the desired 
+result in the preceding match clause. Use modifiers such as `limit`, `sort` and `offset` to retrieve an optionally 
+sorted subset of the matched instances.
+
+The get query triggers a search in a database based on what has been described in the match clause. 
+
+### Syntax
+
+Get queries are written in TypeQL with the following syntax:
+
+```bash
+match pattern [(, pattern)...] 
+[get <variable> [(, <variable>)...];] 
+[sort <variable> [asc|desc];] 
+[offset <value>;] 
+[limit <value>;] 
+[group <variable>;] 
+[count;] [sum|max|min|mean|median|std <variable>;]
+```
+
+Patterns consist of variables and properties for data instances. For more information see the 
+[Query pattern anatomy](03-match.md#query-pattern-anatomy) section.
+
+<div class="note">
+[Note]
+Get clause works like a filter for results. Get query without a get clause will have all variables mentioned in match 
+patterns returned.
+</div>
+
+Aggregation functions must be at the end of the query and do not require a GET clause.
+
+### Variables
+
+A `get` clause is used to specify which variables to include in the results. If omitted, every variable is included.
+
+<!-- test-ignore -->
+```typeql
+match 
+  $p isa person, has name "Kevin Morrison", has email $e;
+get $e;
+```
+
+The example above matches person (`$p`) with a `full-name` attribute value of `Kevin Morrison`, and `email` attribute 
+(`$e`). The `get` clause then filters the results to receive only the `email` (`$e`) attributes.
+
+<!-- test-ignore -->
+```typeql
+match
+  $pe ($x, $y) isa permission;
+  $x isa person, has full-name $x-n;
+  $x-n contains "Kevin";
+  $y (accessed-object: $o, valid-action: $act) isa access;
+  $act has action-name $act-n;
+  $o has path $o-fp;
+get $x-n, $act-n, $o-fp;
+```
+
+The example above uses a `match` clause to do the following:
+
+1. Find `permission` relation (`$pe`) of `$x` and `$y` variables.
+2. Finds `$x` as `person` entities that have `full-name` attribute with value of `$x-n`. 
+3. `$x-n` should contain string `Kevin`.
+4. Finds `access` relation of `$o` as accessed-object and `$act` as valid-action.
+5. `$act` should have `action-name` attribute with value of `$act-n`.
+6. `$o` should have `path` attribute with value of `$o-fp`.
+
+The `get` clause then filters the result from the `match` clause to get only the `person`’s `full-name` (`$x-n`), 
+`action`’s `action-name` (`$act-n`) and `path` of an `object` (`$o-fp`). Every returned result should contain all 
+three variables.
+
+### Modifiers
+
+#### Limit the results
+
+Use the `limit` keyword followed by the maximum number of results to limit the number of results returned.
+
+<!-- test-ignore -->
+```typeql
+match $p isa person; 
+get $p;
+limit 1;
+```
+
+This query returns only one single (and random) instance of type `person`. Consider using `limit` with 
```

Comment:
```suggestion
This query returns only one single (and random) instance of type `person`. We strongly recommend using `limit` with 
```

Reply:
I think I would prefer the original variant here. I think it's closer to the intended message here. Do you think it looks weird or something?